model/src/main/java/cz/muni/fi/pa165/team/TeamRepositoryImpl.java

```diff
@@ -63,6 +63,22 @@ public Team getTeamById(final UUID teamId)
     }
 
     @Override
+    public Team findTeamById(final UUID teamId)
+    {
+        Assert.notNull(teamId, "Cannot search for a null teamId");
+
+        try {
+            return entityManager
```

Comment:
tady by se dala použít ta `getTeamById` a jen odchytit `TeamNotFoundException` a vrátit null, ať se to neduplikuje